_docs/setup/eureka/index.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Eureka
+overview: Instructions for installing the Istio control plane in a Eureka based environment.
+
+order: 20
```

Comment:
are you sure this is the right order? Add +10 after Consul/index.md and then change the order value in index.md for CloudFoundry, Mesos.

Reply:
nice catch